README.md

```diff
@@ -13,38 +13,238 @@
 Creates a Docker container with an installation of the
 [gophish](https://getgophish.com) phishing framework.
 
-## Usage ##
+## Running ##
 
-### Install ###
+### Running with Docker ###
 
-Pull `cisagov/gophish` from the Docker repository:
+To run the `cisagov/gophish` image via Docker:
 
-    docker pull cisagov/gophish
+```console
+docker run cisagov/gophish:0.0.1
```

Comment:
All of these `0.0.1` versions need to be updated to [`0.0.6`](https://github.com/cisagov/gophish-docker/blob/lineage/skeleton/src/version.txt).
```suggestion
docker run cisagov/gophish:0.0.6
```

Reply:
See commit 2405eea.